tests/fixtures/kamereon/vehicle_action/horn-lights.horn.json

```
@@ -0,0 +1,7 @@
+{
+   "data": {
+      "id": "guid",
+      "type": "HornLights",
+      "attributes": {"action": "start", "target": "horn"}
+   }
+}
```

Pre-commit says there should be a trailing carriage return
```suggestion
}

```